spec/avram/operations/save_operation_spec.cr

```diff
@@ -307,6 +308,126 @@ describe "Avram::SaveOperation" do
     end
   end
 
+  describe ".bulk_upsert" do
+    # TODO: Is this possible? How do we write this?
+    it "should only proc one query" do
```

Comment:
@jwoertink @matthewmcgarvey Any thoughts on how we might be able to test this?

Reply:
https://github.com/luckyframework/avram/blob/ac281f490a85283eaf8b74fd23357dd70a6d7f22/spec/avram/queryable_spec.cr#L35-L42